### PR TITLE
Personal models

### PR DESCRIPTION
**Pull request by alexsivkovu**

### CODE REVIEW

File: back-end/common/models/persons.py [hunk 1]
```diff
@@ -17,3 +17,42 @@ class Meta:
...
+        verbose_name = "Person"
+        verbose_name_plural = "Persons"
+
+    def __str__(self):
+        return self.full_name
+
+
+class Personnel(Person):
+    birthdate = models.DateField()
+    photo = models.ImageField(upload_to='', max_length=128)
```
Comment (TmLev) [inline back-end/common/models/persons.py:43]: @gakhromov decide already.

File: back-end/common/models/persons.py [hunk 2]
```diff
@@ -17,3 +17,42 @@ class Meta:
 
     def __str__(self):
         return self.full_name
+
+
+class Relative(Person):
+    relative_list = [('father', 'отец'), ('mother', 'мать'),
+                     ('sister', 'сестра'), ('brother', 'брат')]
```
Comment (TmLev) [inline back-end/common/models/persons.py:24]: TODO for later: export to `TextChoices`.

File: back-end/lms/models/student.py [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
+
+    class Meta:
+        verbose_name = 'Military specialty'
+        verbose_name_plural = 'Military specialties'
+
+
 class Status(models.Model):
-    status = models.CharField(primary_key=True, max_length=20)
+    status_list = [('student', 'обучающийся'), ('graduated', 'выпускник'),
+                   ('deducted', 'отчислен'), ('enrollee', 'абитуриент')]
```
Comment (TmLev) [inline back-end/lms/models/student.py:31]: TODO as above.

File: back-end/lms/views/populate.py [hunk 4]
```diff
@@ -270,27 +266,26 @@ def create_absence_statuses():
 # pylint: disable=(too-many-locals)
 def create_absences(types: dict[str, AbsenceType],
                     statuses: dict[str, AbsenceStatus],
-                    students: dict[str, Student], nearest_day: datetime):
-    date_f = '%Y-%m-%d'
+                    students: dict[str, Student]):
     values = [
         {
-            'date': (nearest_day - timedelta(7)).strftime(date_f),
+            'date': '2020-09-04',
```
Comment (TmLev) [inline back-end/lms/views/populate.py:272]: Restore @gakhromov's `datetime` dates everywhere, please.

**Review by TmLev — CHANGES_REQUESTED** [review]

File: back-end/lms/views/populate.py [hunk 5]
```diff
@@ -467,9 +460,40 @@ def create_achievement_types():
...
+        '751100': 'Защита информационных технологий'
+    }, {
+        '100182': 'Стрелковые, командир стрелкового отделения'
+    }, {
+        '106646-543': 'Разведывательные, разведчик-оператор СБР, ПСНР'
+    }]
+
+    types = {}
+    for value in values:
+        typ, _ = AchievementType.objects.get_or_create(achievement_type=value)
```
Comment (vladisa88) [inline back-end/lms/views/populate.py:489]: Why do you create `AchivementType` object here? I think it should be `MilSpeciality` instead

**Review by gakhromov — APPROVED** [review]
Apart from changing lists to `TextChoices`, everything looks good to me. Don't think we have to change it rn, so lets merge this thing.

**Review by TmLev — APPROVED** [review]

**Pull request merged**